### ISSUE TITLE
SQL logging contains all variables

### ISSUE THREAD

**Issue by Bertg** [janko/sequel-activerecord_connection#17]
I just started using this gem in the context of [rodauth-rails](https://github.com/janko/rodauth-rails/) and it works pretty well.

However, I did find an issue with how SQL query logs are being processed. 

A direct ActiveRecord query is generally logged as follows:

```SQL
SELECT "accounts".* FROM "accounts" WHERE "accounts"."email" = $1 LIMIT $2
```

Where the Sequel log generated by Rodauth looks like this:

```SQL
SELECT * FROM "accounts" WHERE ("email" = 'clint@pacocha.org') LIMIT 1
```

The Sequel log is problematic, as the user's email address is present and thus causes issues with storing that in a central location because of security and legal considerations. It also makes it a lot harder to find "the same" queries for performance monitoring.

These queries are collected through the ActiveSupport event system. I'm not sure if this is even the correct project to submit this bug to. This could [sequel](https://github.com/jeremyevans/sequel) or [rodauth](https://github.com/jeremyevans/rodauth) issue, possibly even a postgres specific issue. Any help here would be greatly appreciated.

For completions these are the relevant gems in use:

```
rodauth (2.17.0)
rodauth-rails (0.16.0)
sequel (5.48.0)
activerecord (6.1.4.1)
activesupport (6.1.4.1)
lograge (0.11.2)
lograge-sql (1.3.1)
```

**Comment by janko** [comment]
@jeremyevans Do you have an idea how to mitigate this with Sequel/Rodauth in general? Is the only option to use bound variables, like Active Record uses?

**Comment by janko** [comment]
It seems that the sql_log_normalizer DB extension was working with this gem only if a logger was set on the Sequel DB object (which is rarely useful, since we're already using Active Record logging). I now released version 1.2.5 which should make it work even when no logger is set.
